### ISSUE TITLE
In-line commit notes

### ISSUE THREAD

**Issue by matthew-white** [jcabi/jcabi-github#1004]
Is there support for [in-line commit notes](https://github.com/blog/622-inline-commit-notes) outside pull requests? Perhaps `Commit.Smart` could get a `comments()` method along the lines of `Issue.Smart`? I'm trying to write a client that helps users search in-line notes.

**Comment by yegor256** [comment]
@matthew-white you are talking about this: https://developer.github.com/v3/pulls/comments/ If not, can you find that "in-line notes" in Github API: https://developer.github.com/v3/

**Comment by matthew-white** [comment]
I think it's this: https://developer.github.com/v3/repos/comments/

**Comment by yegor256** [comment]
Right, we don't have access to them. We should implement `Repo.comments()` method and `RepoComments` interface. Will do it soon.
